target/linux/ramips/files-4.14/drivers/net/ethernet/mtk/mtk_offload.c

```diff
@@ -173,6 +180,10 @@ int mtk_flow_offload(struct mtk_eth *eth,
 		goto write;
 	}
 
+        if(mtk_check_hashcollision(eth, ohash))       // Two-way hash: when hash collision occurs, the hash value will be shifted to the next position.
+                ohash += 1;
+        if(mtk_check_hashcollision(eth, rhash))
+                rhash += 1;
 	mtk_foe_set_mac(&orig, dest->eth_src, dest->eth_dest);
```

Comment:
Thanks for the PR. I am wondering what happens if +1 is also populated ? should we maybe add a loop instead and iterate until +N is free ?